logstash-core/lib/logstash/util/wrapped_acked_queue.rb

```diff
@@ -332,10 +332,6 @@ def flush_signal_received?
 
       private
 
-      def iterating?
-        @iterating
-      end
```

Comment:
@guyboertje hmm is it worth the noise for this in general though? I mean here, I went for this because it's a very hot loop and anything that makes the bytecode that is generated more predictable is a win, but in something like the settings class it doesn't really matter?